lib/tapioca/dsl/helpers/active_record_column_type_helper.rb

```diff
@@ -121,8 +121,8 @@ def column_type_for(column_name)
           end
         end
 
-        sig { params(column_type: T.untyped).returns(String) }
-        def type_for_activerecord_value(column_type)
+        sig { params(column_type: T.untyped, null: T::Boolean).returns(String) }
+        def type_for_activerecord_value(column_type, null:)
```

Comment:
```suggestion
        def type_for_activerecord_value(column_type, column_nullability:)
```

Something like this will be better.